File: server .Net/EmployeeServer/EmployeeServer/Model/EmployeePostModel.cs
﻿using EmployeeServer.Core.Entities;

namespace EmployeeServer.Api.Model
{
    public class EmployeePostModel
    {
        //public int Id { get; set; }
        public string Identity { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime DateOfBirth { get; set; }
        public Gender Gender { get; set; }
        //public List<Role> Role { get; set; }
        public List<EmployeeRolePostModel> EmployeeRoles { get; set; } // הוספה
        //public bool Status { get; set; }
    }
}

File: EmployeeClient/src/app/classes/postModel/employee.postModel.ts

import { EmployeeRolePostModel } from "./employeeRole.postModel"

export class EmployeePostModel{
    // id!:number
    identity!:string
    firstName!:string
    lastName!:string
    startDate!:Date
    dateOfBirth!:Date
    maleOrFmale!:boolean
    employeeRoles!:EmployeeRolePostModel[]
    status!:boolean
}